src/decorators/index.ts

```diff
@@ -22,3 +22,4 @@ export {
 	Next,
 } from './base-param-decorators.class';
 export { Log, Logger } from './log.decorator';
+export {} from './symbols';
```

Comment:
dont you need to define the 
```suggestion
export {
  endpointMetaSymbol,
  paramMetaSymbol,
  controllerMetaSymbol,
  loggerMetadataKeySymbol,
  middlewareControllerKey,
  directMiddlewareSymbol,
  preExecutionHookSymbol,
  afterResponseExecutionHookSymbol,
} from './symbols';
```